documents/motions/08.2020.2 - Trainee Delegates.md

```diff
@@ -0,0 +1,52 @@
+# SUBMISSION OF PROPOSED MOTION
+
+**Motion number:** 8.2020.2  
+**Subject:** WCA Trainee Delegates  
+**Intent:** Procedure of acknowledgement of WCA Trainee Delegates and description of their rights and duties  
+**Submitted by:** Board of Directors  
+**Date:** January 31, 2020  
+
+# Motion
+
+Trainee Delegates are Community Members who have been appointed by the WCA to train for the role of WCA Delegate.
+
+1. Trainee Delegates will require a time period of extra mentoring and assistance by Senior Delegates and/or Regional Delegates, to support their development to Candidate Delegates and Staff of the WCA.
```

Comment:
Should this extra mentoring also be reflected in the Senior Delegates motion (e.g. in 3.2.3) and Regional Delegates motion? I would also like to see some of this mentoring be codified as the responsibility of all non-Trainee Delegates, since that is the purpose of requiring another Delegate. A Senior/Regional Delegate won't always be at the competition with the Trainee, so some of the responsibility of training them should fall to the Candidate/Full Delegate present.

Reply:
Clarified this in the upcoming amendments.